assets/css/styles.css

```diff
@@ -108,3 +108,9 @@ img {
   background-color: lightgreen;
   color: darkgreen;
 }
+
+.Visitor{
+  width: min-content !important;
+  max-width: 100%;
+  margin: auto;
+}
```

Comment:
Can you add a new line at the bottom of the styles.css file @divya-nshi

Reply:
@YoshithaRathnayake i have added it